samples/attestation/CMakeLists.txt

```diff
@@ -3,6 +3,19 @@
 
 cmake_minimum_required(VERSION 3.11)
 
+# If the CC environment variable has been specified or if the CMAKE_C_COMPILER
```

Comment:
Can this be abstracted into a cmake utility provided by the SDK?
Note: We can to that in next release too.

Reply:
There is a trade-off; we could make the compiler selection transparent or provide a cmake utility that does this. However, the later may require additional `include()` for the cmake script. We can't make the include as part of `OpenEnclave` package because we invoke `find_package(OpenEnclave ...)` after `project`. I find that the current approach is simpler.  What do you think?

Reply:
Note: we don't need this PR for this release as the clang-8 support is independent from the samples.

Reply:
Sounds good. We can address this the next time we upgrade compiler.